src/components/SettingsModal/SettingsStorage.js

```diff
@@ -31,9 +33,34 @@ export function getDefaultSettings() {
  */
 export async function getSettingsValue(key) {
   const value = await investSettingsStore.getItem(key);
+  if (!value) {
+    return getDefaultSettings()[key];
```

Comment:
Hey Dave, I'm just curious how you found this necessary? Is this just to avoid a case where we've somehow corrupted the settings object?

Reply:
I'm not sure it strictly is necessary, though these settings are never meant to undefined, so if somehow they are, I think it's good that `App` never knows that and always gets valid values to render. 

Anyway this is actually the same logic that was already in place. It used to be inside the `app.jsx` componentDidMount. I pulled it out of there so we never have to think about the internals of `SettingsStorage` when we're dealing with `app.jsx`. 

Here's those changes: https://github.com/natcap/invest-workbench/pull/143/files/1833d7c1b42aabad24befd6d31da372145bdfbf2#diff-73ba1bb45657b12f529b48ba4515bc802efe0eb6184f56c364c6f5c9753b7441R57